### DISCUSSION TITLE
Download all images from a feed when fetching feeds in the background

### DISCUSSION THREAD

**Discussion by Terkyz** [spacecowboy/Feeder#239] [Ideas]
Sometimes I find with feeds that are basically based in an image and I can't see the image because I don't have internet connection and Feeder only fetch the text, so I think it would be useful that Feeder automatically fetch all the images when fetching feeds (and the user is on WiFi).

When you manually enter Feeder and start scrolling through feeds, the feed image (if there is) generally is downloaded, I meant this behaviour but while the app is fetching feeds in the background.

**Comment by spacecowboy** [comment]
Something I've considered in the past but never acted on.

**Comment by Terkyz** [comment]
I hope you have the time and desire to implemented soon!
